app/src/main/java/com/poti/android/presentation/user/editprofile/EditProfileScreen.kt

```
@@ -0,0 +1,65 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.poti.android.R
+import com.poti.android.core.designsystem.component.field.PotiShortTextField
+import com.poti.android.core.designsystem.component.navigation.PotiHeaderPage
+import com.poti.android.presentation.user.component.EditableUserProfileImage
+
+@Composable
+fun EditProfileRoute(modifier: Modifier = Modifier) {
+}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)